.github/workflows/build.yml

```
@@ -485,8 +487,12 @@ jobs:
         uses: dtolnay/rust-toolchain@nightly
         with:
           components: clippy
-      # do not deny warnings, as nightly clippy sometimes has false negatives
-      - run: ./admin/clippy
+      # Do not deny warnings, as nightly clippy sometimes has false negatives.
+      # - Ignore `clippy::disallowed_types` as a workaround since `disallowed_types` configured in `.clippy.toml`
+      #   is only intended for main `rustls` crate.
+      - run: ./admin/clippy -- --allow clippy::disallowed_types # --allow clippy::incompatible_msrv
+      # Check the main crate for all Clippy nightly warnings; do not deny.
```

```suggestion
      # Check the main crate for all Clippy nightly warnings, but do not deny them.
```

(another nit, I think this would give us slightly better consistency.)